sequencer/src/bin/espresso-dev-node.rs

```diff
@@ -188,7 +188,11 @@ async fn main() -> anyhow::Result<()> {
         .network_config(network_config)
         .build();
 
-    let network = TestNetwork::new(config, MockSequencerVersions::new()).await;
+    let network = TestNetwork::new(
+        config,
+        SequencerVersions::<SequencerApiVersion, MarketplaceVersion>::new(),
```

Comment:
Do we want to run the dev node with the marketplace version or have an argument to run v1/fee/marketplace?

Reply:
I am not sure. dev-node is for the developers who want to integrate their rollups with espresso. If you think they have to use this function, then it will be necessary for us to add this function.

Reply:
I think we should support any version. Marketplace will not be live for some time.

Reply:
This was discussed in standup and I think we only need marketplace version here cc @sveitser

Reply:
Ok. I think it limits the usefulness of devnode, but I also don't remember the conversation. I go ahead a approve and if we need it we can add versions later.

Reply:
I do agree tho that we should have support for pre marketplace version so I'll take a look at it next week. I think this blocks some other work so should be merged

Reply:
https://github.com/EspressoSystems/espresso-sequencer/issues/2033